### ISSUE TITLE
brows mode does not work in google chrome

### ISSUE THREAD

**Issue by abdullah5490** [nvaccess/nvda#5772]
i don't know why is it happening, but i'm unable to use google chrome due to this problem. NVDA just keep saying unknown unknown when i tried to brows a webpage in google chrome. its chrome 48 with NVDA2016.1 same issue was present with 2015.4. any clue?

**Comment by leonardder** [comment]
Can you still reproduce this? Chrome is working just fine here, except for the odd legacy windows you get sometimes

**Comment by abdullah5490** [comment]
no, @leonardder its fine now in usage of brows mode. but that legacy windows is remain as problem now. and, while browsing the web, the virtual curser move at the beginning of the page everytime whenever I exit from a edit field etc.

**Comment by leonardder** [comment]
Ok. Closing this issue as works for me. Please find or open other issues for the other issues you're having. I belief there's at least one issue for the legacy windows, but this needs a fix by Google.
